src/utils/utils.js

```diff
@@ -20,6 +20,8 @@ export const parseUrl = url => {
   }
 };
 
+export const parseNumber = value => (!Number.isNaN(parseFloat(value)) && !Number.isNaN(Number(value)) ? Number(value) : null);
```

Comment:
we can reuse the result of Number(value)

Reply:
fixed